core/pom.xml

```diff
@@ -78,6 +78,12 @@
 			<artifactId>logback-classic</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>uk.org.webcompere</groupId>
+			<artifactId>system-stubs-jupiter</artifactId>
+			<version>2.0.1</version>
```

Comment:
can you please move version from both modules to parent pom and define dependency in dependencyManagement.